predictions.py

```
@@ -1,232 +1,267 @@
-    # Run predictions and save to CSV
-    run_predictions(struct_dir, model, args.output, gonames, goterms)
+import torch
+from pathlib import Path
+import os
+import numpy as np
+import argparse
+import glob
+import csv
+from transformers import BertTokenizer, BertModel
+import scipy.sparse as sp
+from model import GCN, RareLabelGNN
```

Ruff (0.8.2)

`model.RareLabelGNN` imported but unused

Remove unused import: `model.RareLabelGNN`

(F401)